app.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+function CookieStand(location, min, max, avg) {
+  this.location = location;
+  this.min = min;
+  this.max = max;
+  this.avg = avg;
+  this.cookies = [];
+}
+
+CookieStand.prototype.getCookiesSold = function() {
+  for(var i = 0; i < 15; i++) {
+    this.cookies.push(Math.floor(((Math.random() * (this.max - this.min + 1)) + this.min) * this.avg));
+  }
+  return this.cookies;
+};
+
+CookieStand.prototype.getTotalCookies = function() {
+  var total = 0;
+  for(var i = 0; i < this.cookies.length; i++) {
+    total += this.cookies[i];
+  }
+  return total;
+};
+
+CookieStand.prototype.getTableRow = function() {
+  var row = document.createElement('tr');
+  row.setAttribute('class', 'sales-row');
+  var td = document.createElement('td');
+  td.setAttribute('class', 'location-td');
+  td.textContent = this.location;
+  row.appendChild(td);
+  for(var i = 0; i < this.cookies.length; i++) {
+    td = this.getTableData(i);
+    row.appendChild(td);
+  }
+  td.textContent = this.getTotalCookies();
+  row.appendChild(td);
+  return row;
+};
+
+CookieStand.prototype.getTableData = function(index) {
+  var td = document.createElement('td');
+  td.setAttribute('class', 'sales-td');
+  td.textContent = this.cookies[index];
+  return td;
+};
+
+function createTableHeaderRow() {
+  var row = document.createElement('tr');
+  row.setAttribute('id', 'sales-header-row');
+  var th = document.createElement('th');
+  th.setAttribute('class', 'sales-th');
+  th.textContent = '';
+  row.appendChild(th);
+  var time;
+  for(var i = 1; i < 15; i++) {
+    th = document.createElement('th');
+    th.setAttribute('class', 'sales-th');
+    time = i + 5;
+    if(time < 12){
+      th.textContent = time + ':00am';
+    } else if(time > 12){
+      time -= 12;
+      th.textContent = time + ':00pm';
+    } else {
+      th.textContent = time + ':00pm';
+    }
+    row.appendChild(th);
+  }
+  th = document.createElement('th');
+  th.setAttribute('class', 'sales-th');
+  th.textContent = 'Daily Location Total';
+  row.appendChild(th);
+  return row;
+}
+
+function createTableFooterRow(pike, seaTac, seattleCenter, capitolHill, alki) {
```

Comment:
this is a creative way to do it, but what if you're updating the number of stores you own?